pkg/api/server.go

```diff
@@ -28,12 +28,12 @@ import (
 )
 
 var version string = "0.0.0"
```

Comment:
`pkg/api/server.go:30:13: should omit type string from declaration of var version; it will be inferred from the right-hand side`
```suggestion
var version = "0.0.0"
```